d2renderers/d2svg/d2svg.go

```diff
@@ -114,6 +114,9 @@ func arrowheadDimensions(arrowhead d2target.Arrowhead, strokeWidth float64) (wid
 	case d2target.DiamondArrowhead:
 		widthMultiplier = 11
 		heightMultiplier = 9
+	case d2target.FilledCircleArrowhead, d2target.CircleArrowhead:
+		widthMultiplier = 14
```

Comment:
I'd expect these to be equal for circles.
If they don't need to be equal, they are the same values as for `Cfone`, `CfMany`, so you could join the `case`

Reply:
I was looking at joining the cases when I wrote it, but I felt as if that was too much shoving into one case. If at some point later things needed to be broken out or split apart it would be a bit more mental overhead, also I felt like putting the distinct shapes together as I had seen above made more sense then putting two separate shapes into the same case.  I do agree it is sloppy in some way. I was not sure what would be the better option.

Reply:
Cases joined for now.